node/node.go

```diff
@@ -27,6 +28,13 @@ type Node struct {
 
 	// CoreClient provides access to a Core node process.
 	CoreClient core.Client `optional:"true"`
+
+	// BlockService provides access to the node's Block Service
+	// TODO @renaynay: I don't like the concept of storing individual services on the node,
```

Comment:
Why?

Reply:
Because many will be `nil` for `Light` nodes (e.g. BlockService).

Reply:
There is no technical issue here. We can come up with something later to make this prettier, but for now, that just works.